forage-android/src/main/java/com/joinforage/forage/android/network/data/CapturePaymentRepository.kt

```diff
@@ -90,7 +91,8 @@ internal class CapturePaymentRepository(
         contentId: String,
         paymentRef: String
     ): ForageApiResponse<String> {
-        var attempt = 1
+        var attempt = 0
```

Comment:
technically, if you change `attempt` from `1` -> `0`, the if-statement `if (attempt == MAX_ATTEMPTS) { ... }` will get triggered after `MAX_ATTEMPTS + 1`. Is this OK? If we are making any gaurantees to customers that a timeout will happen in `pollingInterval * MAX_ATTEMPTS` or sooner than we may want to change the if-statement below to be `if (attempt == MAX_ATTEMPTS - 1) { ... }`.

IMO I think it would be easier to read if the if-statement remained `if (attempt == MAX_ATTEMPTS) { ... }` and we updated this code:
```kotlin
var intervalTime: Long = if (attempt < pollingIntervals.size) {
    pollingIntervals[attempt]
} else { ... }
```

to be 

```kotlin
val index = attempt - 1
var intervalTime: Long = if (index < pollingIntervals.size) {
    pollingIntervals[index]
} else { ... }
```

Reply:
That makes sense, thanks for the good catch! I will use the `var index = attempt - 1` approach